config/app.php

```diff
@@ -166,8 +166,9 @@
      * - `log` - boolean - Whether or not you want exceptions logged.
      * - `exceptionRenderer` - string - The class responsible for rendering
      *   uncaught exceptions. If you choose a custom class you should place
-     *   the file for that class in src/Error. This class needs to implement a
-     *   render method.
+     *   the file for that class in src/Error. This class needs to implement a `render()` method.
+     *   `errorRenderer` - string - The class responsible for rendering PHP errors. This class needs
```

Comment:
We should mention the custom renderer is used for both cli and non-cli environments.